src/components/camera.js

```diff
@@ -1,6 +1,7 @@
 var registerComponent = require('../core/component').registerComponent;
 var THREE = require('../lib/three');
 var utils = require('../utils/');
+var bind = utils.function.bind;
```

Comment:
I'm not sure we will need more functions other than `bind`. Why optimizing for something we are not sure that will ever happen. `function` is a such a generic name that the namespace does not add any information. it just feels like clutter.